packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -304,15 +304,29 @@ export = {
     },
   },
 
-  'can import a virtual router'(test: Test) {
+  'Can import Virtual Routers using ARN and attributes'(test: Test) {
```

Comment:
Same comment here - leave the old test as-is, and create a new one for `fromVirtualRouterAttributes `.